v1/faasd.yaml

```diff
@@ -0,0 +1,50 @@
+description: A portable FaaS framework
+version: 0.11.4
+
+instances:
+  faasd:
+    limits:
+      min-cpu: 2
+      min-mem: 2G
+      min-disk: 20G
+    cloud-init:
+      vendor-data: |
```

Comment:
It'd be preferable for you to source this from our repo instead of maintaining it in two places?

Reply:
I'm not sure that question makes sense.

We use cloud-init for setting up VMs on various clouds and IaaS platforms, and various terraform modules also.

The openfaas/faasd repo is the source of truth, can you have your template / app definition pull it from there?

Reply:
That's what I was asking - whether you use it elsewhere. At the moment we have no way to pull in metadata from the outside. We'll have to see how we could integrate that.